File: 2021/15. UI. UIView, CALayer, UIWindow/Demos/Demo 1/Demo/RSView.m
//
//  RSView.m
//  Demo
//
//  Created by Viktar Semianchuk on 3/17/20.
//  Copyright © 2020 Viktar Semianchuk. All rights reserved.
//

#import "RSView.h"

@implementation RSView

-(instancetype)initWithFrame:(CGRect)frame andColor:(UIColor *)color
{
    self = [super initWithFrame:frame];
    if (self) {
        self.backgroundColor = color;
    }
    return self;
}

@end

File: 2021/15. UI. UIView, CALayer, UIWindow/Demos/Demo 1/Demo Swift/Demo Swift/RSView.swift
//
//  RSView.swift
//  Demo Swift
//
//  Created by Andrew Pokachailo on 6/28/21.
//

import UIKit

class RSView: UIView {
    
    init(withFrame frame: CGRect, andColor color: UIColor) {
        super.init(frame: frame)
        self.backgroundColor = color
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
    /*
    // Only override draw() if you perform custom drawing.
    // An empty implementation adversely affects performance during animation.
    override func draw(_ rect: CGRect) {
        // Drawing code
    }
    */

}
